rdkit/Chem/UnitTestMol3D.py

```diff
@@ -3,18 +3,47 @@
 """unit testing code for 3D stuff
```

Comment:
This contains the tests for the GetBestRMS and GetConformerRMS, so in this respect it belongs to the PR. The changes are however for the TorsionFingerprint (as the branch is named wrongly).